.github/workflows/chromatic.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         run: npm ci
         # 👇 Adds Chromatic as a step in the workflow
       - name: Publish to Chromatic
-        uses: chromaui/action@v1
+        uses: chromaui/action@v11.4.0
```

Comment:
Chromatic released v11.5 of their cli tool which updated the action too. It forces node 20 and some incompatibility with older storybook versions...
https://github.com/chromaui/chromatic-cli/releases/tag/v11.5.0